Add instructions on how to request an RRA (#81)
mozilla/infosec.mozilla.org: docs/guidelines/risk/rapid_risk_assessment.md

The RRA risk table facilitates discovering the answers to these questions.

> # How-to: Request an RRA be run on your service

Reviewer: ```suggestion
# How-to: Request an RRA be run for you service
```

Reviewer 2: I feel "Request an RRA of your service" makes more sense here when considering the expanded acronym — "Request a Rapid Risk Analysis of your service".

> (File a bug)[https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=Rapid%20Risk%20Analysis] to start an RRA. Include basic information about the project, including any relevant links and 0-2 additional people to include during the meeting. Our team will automatically triage the RRA and someone from InfoSec will reach out to you.

Reviewer: ```suggestion
To manually request an RRA, please (file a bug)[https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=Rapid%20Risk%20Analysis] to our component. Please include basic information about the project, a diagram, any relevant links and 0-2 additional people to invite during for the assessment.
```

> Note that any projects started through CASA will be automatically evaluated, and bugs will be automatically filed to start the RRA.

Reviewer: ```suggestion

```
